src/internal_events/file.rs

```diff
@@ -234,6 +237,8 @@ mod source {
         }
     }
 
+    const WATCH_FAILED: &str = "watch_failed";
```

Comment:
I think all of these could get moved into `error_type` even if they are only used in one module.

Reply:
as mentioned previously, IMO the `watch_field` should be the `error` value and the `error_type` should be something like `READER_FAILED` in order not to put the error value as a string in the `error` field to reduce the cardinality.